hassio/src/system/hassio-supervisor-info.ts

```diff
@@ -249,6 +273,32 @@ class HassioSupervisorInfo extends LitElement {
     });
   }
 
+  private async _unsupportedDialog(): Promise<void> {
+    const resolution = await fetchHassioResolution(this.hass);
+    await showAlertDialog(this, {
+      title: "You are running an unsupported installation",
+      text: html`Below is a list of issues found with your installation, click
+        on the links to learn how you can resolve the issues. <br /><br />
+        <ul>
+          ${resolution.unsupported
+            .filter((issue) => issue in ISSUES)
```

Comment:
this works, but we now go over it twice, better to just return an empty string in the `map` (or maybe we should print the key if we don't have a description, instead of an empty list?)

Reply:
So instead of checking it twice total, I should add a check inside the map, going over it up to 10 times if all is wrong?
Wouldn't that be worse?

Reply:
Yup, didn't think right :smile: